### ISSUE TITLE
About learning rate setting

### ISSUE THREAD

**Issue by IceyWuu** [xbchen82/SRGAT#1]
The paper said "The learning rate is set to 10−4 initially and decreases to half every 200 epochs. ", but I found that in your codes, in the optimizer setting, also set T_max = self.args.num_epochs, which is also 200. Is it should be T_max=1000? Equal to the total epoch number.
